nolearn/lasagne/base.py

```diff
@@ -83,6 +83,23 @@ def __getstate__(self):
         return state
 
 
+def train_split(X, y, net, eval_size=0.2):
```

Comment:
Why not make this a class rather than a function so that eval_size can't be closed over rather than being passed in on each split.